README_fdmdv.md

```diff
@@ -0,0 +1,122 @@
+# README_fdmdv
+
+## Introduction
+
+A 1400 bit/s (nominal) Frequency Division Multiplexed Digital Voice (FDMDV) modem based on [1].  Used for FreeDV 1600.
```

Comment:
Purely a stylistic thing, but I like making link anchors contain their content rather than a more generic _click here_, even if it is redundant.  So I might have written:

```markdown
A 1400 bit/s (nominal) Frequency Division Multiplexed Digital Voice (FDMDV) modem based on the [FreeDV 1600 Specification](https://freedv.org/freedv-specification), as implemented in FreeDV 1600.
```